docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+
+services:
+    db:
+        image: postgres
+        env_file: .env
+        volumes:
+          - pgdata:/var/lib/posgresql/data
```

Comment:
Should be: ```/var/lib/pos**t**gresql/data```?

Even after fixing I get stuck at:

```
db_1             | running bootstrap script ... ok
db_1             | performing post-bootstrap initialization ... 2020-03-24 14:28:54.434 UTC [40] FATAL:  zero-length delimited identifier at or near """" at character 12
db_1             | 2020-03-24 14:28:54.434 UTC [40] STATEMENT:  ALTER USER ""postgres"" WITH PASSWORD E'"fedireads"';
db_1             | 	
db_1             | child process exited with exit code 1
db_1             | initdb: removing contents of data directory "/var/lib/postgresql/data"
fedireads_db_1 exited with code 1
```

Reply:
after my last commit that fixes this, I re-ran `docker-compose build` and `docker-compose up` and this error went away for me🤞

Reply:
I also changed my database type in .env from sqlite to postgres, which may be related

Reply:
Yeah, it's working for me too now. :)